spec/timezone.html

```diff
@@ -232,7 +240,16 @@ <h1>Temporal.TimeZone.prototype.getPossibleInstantsFor ( _dateTime_ )</h1>
         1. Let _timeZone_ be the *this* value.
         1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimezone]]).
         1. Set _dateTime_ to ? ToTemporalDateTime(_dateTime_).
-        1. <mark>TODO.</mark>
+        1. If _timeZone_.[[IsOffset]] is *true*, then
+          1. Let _epochNanoseconds_ be ! GetEpochFromParts(_dateTime_.[[ISOYear]], _dateTime_.[[ISOMonth]], _dateTime_.[[ISODay]], _dateTime_.[[ISOHour]], _dateTime_.[[ISOMinute]], _dateTime_.[[ISOSecond]], _dateTime_.[[ISOMillisecond]], _dateTime_.[[ISOMicrosecond]], _dateTime_.[[ISONanosecond]]).
+          1. Let _instant_ be ? CreateTemporalInstant(_epochNanoseconds_ − _timeZone_.[[OffsetNanoseconds]]).
+          1. Return ? CreateArrayFromList(« _instant_ »).
+        1. Let _possibleEpochNanoseconds_ be ? GetIANATimeZoneEpochValue(_timeZone_.[[Identifier]], _dateTime_.[[ISOYear]], _dateTime_.[[ISOMonth]], _dateTime_.[[ISODay]], _dateTime_.[[ISOHour]], _dateTime_.[[ISOMinute]], _dateTime_.[[ISOSecond]], _dateTime_.[[ISOMillisecond]], _dateTime_.[[ISOMicrosecond]], _dateTime_.[[ISONanosecond]]).
+        1. Let _possibleInstants_ be the empty List.
```

Comment:
Nit: "a new empty List" or "« »". (Unlike strings, lists are mutable, so "the" empty list doesn't make sense.)